portugues/colabora/comdiferencas/diff/pull/reenvio/watch/emprego/emprego.md

```diff
@@ -1,4 +1,13 @@
 Você foi aprovado em uma entrevista de emprego na empresa cujo nome possui três letras
  pelo fato de ter contribuido para a comunidade open source, através de correções de bugs 
-e criação de novas features em aplicações de terceiros:
```

Comment:
Yes, change the colon to a period was necessary to improve readability and respect the Portuguese grammar.